vertica_python/vertica/connection.py

```diff
@@ -292,8 +292,9 @@ def startup_connection(self):
         user = self.options['user'].encode(ASCII)
         database = self.options['database'].encode(ASCII)
         password = self.options['password'].encode(ASCII)
+        label = self.options['label'].encode(ASCII) if 'label' in self.options else None
```

Comment:
`label` is a new connection setting, that should be documented into README.

Reply:
It would make more sense if you rename it to `session_label`. This is also the key name used in JDBC.